src/AppBundle/Controller/ComponentController.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+	namespace AppBundle\Controller;
+
+
+	use AppBundle\Entity\Component;
+	use AppBundle\Form\ComponentType;
+	use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+	use Symfony\Component\HttpFoundation\Request;
+	use Ob\HighchartsBundle\Highcharts\Highchart;
+
+
+	class ComponentController extends Controller
+	{
+		/***
+		* @param $dashboardId
+		* @return \Symfony\Component\HttpFoundation\RedirectResponse
+		*/
+		public function addComponentAction($dashboardId)
+		{
+			$component = new Component();
+
+			$em = $this->getDoctrine()
+					->getManager();
+
+			$component->setDashboard($em->getRepository('AppBundle:Dashboard')->find($dashboardId));
+
+			$em->persist($component);
+			$em->flush();
+
+
+
+			return $this->redirectToRoute('app_edit_component', array('componentId' => $component->getId()));
+			
+		}
+
+		/***
+		* @param Request $request
+		* @param $componentId
+		* @return \Symfony\Component\HttpFoundation\RedirectResponse|\Symfony\Component\HttpFoundation\Response
+		*/
+		public function editComponentAction(Request $request, $componentId)
+		{
+			//Return table and field list (bypass doctrine)
+			$conn = $this->get('database_connection');
+			$tables = $conn->fetchall('SHOW TABLES');
+
+			foreach($tables as $table)
+			{
+				foreach($table as $key => $tab)
+				{
+					//Exclude the Sqldashboard's tables
+					$test = explode('_', $tab);
+					if ($test[0] != "sqldashboard" && $tab != "dashboard_user")
+					{
+						//Display the tables
+						$field = $conn->fetchall('DESCRIBE '.$tab);
+						$listTables[$tab] = $field;
+					}
+				}
+			}
+			
+			//Change the head title
+			$mainTitle = "Edit component";
```

Comment:
Don't do this, you have to put texts in resources files http://symfony.com/doc/current/translation.html